core/src/dsnp/api_types.rs

```diff
@@ -1,8 +1,14 @@
 use crate::dsnp::{dsnp_configs::KeyPairType, dsnp_types::DsnpUserId};
-pub use dsnp_graph_config::{ConnectionType, PrivacyType};
+pub use dsnp_graph_config::{ConnectionType, PageId as PageIdType, PrivacyType};
 use dsnp_graph_config::{GraphKeyType, SchemaId};
 use std::{cmp::Ordering, fmt::Debug};
 
+/// Graph page id
+pub type PageId = PageIdType;
+
```

Comment:
I think if you just do `pub use dsnp_graph_config::PageId` it will re-export; you don't need to redefine the `PageId` type again.